students/KevinWaldron/lesson02/print_grid.py

```diff
@@ -0,0 +1,10 @@
+def print_grid(size, cell):
+    """ Prints a grid with the given size (rows / columns) and cell size """
+    rowStr = ("+" + (" -" * cell) + " ") * size + "+"
+    colStr = ("|" + (2 * cell * " ") + " ") * size + "|"
```

Comment:
per pep8 (python style guidelines) var names should be snake_case -> `row_str`, `col_str`, this also applies to modules (file names) and function names.